lib/vfscore/main.c

```diff
@@ -401,7 +401,7 @@ UK_LLSYSCALL_R_DEFINE(ssize_t, pread64, int, fd,
 	};
 	ssize_t bytes;
 
-	bytes = uk_syscall_r_preadv((long) fd, (long) &iov,
+	bytes = uk_syscall_r_pread64((long) fd, (long) &iov,
```

Comment:
This does not make much sense to me. `UK_LLSYSCALL_R_DEFINE()` defines here `uk_syscall_r_pread64`. This change will end up in an endless recursion.